<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assistantinfo</class>
 <widget class="QDialog" name="assistantinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_assistant">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>441</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_assistant_load_tbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LOAD TO TABLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_assistan_blood_group_update">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Blood Group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_assistant_blood_group"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_third_status">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
